src/components/landing/Projects/index.jsx

```diff
@@ -6,78 +6,78 @@ import Star from 'components/common/Icons/Star';
 import Fork from 'components/common/Icons/Fork';
 import { Wrapper, Grid, Item, Content, Stats, Languages } from './styles';
 
-export const Projects = () => {
```

Comment:
try to put back this file and see if the build passes in ci cd (I know it fails locally but in ci cd it should work perhaps )

Reply:
I follow your saying(uncomment that part and build in ci cd) but it failed again)